frontend/src/static/css/v_authorship.scss

```diff
@@ -100,6 +108,7 @@
       &.active {
         pre {
```

Comment:
GitHub's colour for authored lines is `#e6ffed`

Reply:
Okay, can change this